ckanext/switzerland/helpers/frontend_helpers.py

```diff
@@ -2,22 +2,21 @@
 helpers belong in this file if they
 are used in frontend templates
 """
+import json
+import logging
 from collections import OrderedDict
-import ckan.plugins.toolkit as tk
+
+import ckan.lib.i18n as i18n
```

Comment:
I know, most of the developers know the meaning of i18n, but would it be helpful to use a more "speaking" term like internationalization_toolkit. But in this case, it is also okay to let it like this..